apps/web-giddh/src/app/verify-subscription-transfer-ownership/verify-subscription-transfer-ownership.component.html

```diff
@@ -40,17 +40,50 @@ <h2 class="modal-title">Subscription Transfer</h2>
                 </p>
             </div>
             <div class="modal-footer">
+                <button aria-label="yes" mat-stroked-button color="primary" (click)="onSubmit($event)">
+                    Accept Subscription
+                </button>
+                <button aria-label="no" mat-stroked-button (click)="onReject($event)">Decline Subscription</button>
+            </div>
+        </div>
+    </div>
+</ng-template>
+
+<!--Modal for Transfer Ownership -->
+<ng-template #rejectConfirmation>
+    <div (click)="$event.stopPropagation()">
+        <div>
+            <div class="modal-header">
+                <h2 class="modal-title">Reject</h2>
                 <button
-                    aria-label="yes"
-                    mat-stroked-button
-                    color="primary"
-                    (click)="onSuccess($event)"
+                    aria-label="cross"
+                    mat-icon-button
+                    class="close"
+                    data-dismiss="modal"
+                    (click)="closeReject($event)"
                 >
-                    Accept Subscription
+                    ×
                 </button>
-                <button aria-label="no" mat-stroked-button (click)="closeAllDialogs()" >
-                    Decline Subscription
+            </div>
+            <div class="modal-body">
+                <form [formGroup]="rejectForm">
+                    <div class="w-50">
+                        <input-field
+                            [name]="'reason'"
+                            [label]="Reason"
+                            [placeholder]="'Enter reason'"
+                            [type]="'text'"
+                            formControlName="reason"
+                        >
+                        </input-field>
+                    </div>
+                </form>
+            </div>
+            <div class="modal-footer">
+                <button aria-label="yes" mat-stroked-button color="primary" (click)="onSubmit($event, true)">
+                    Submit
```

Comment:
Translation over all file

Reply:
Nisha will be working on translation and designing
CC : Confirmed with Kriti